package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "genesys-spark-components",
-  "version": "4.64.0",
+  "version": "4.64.1",
```

Comment:
Again, no need to update this. Once a change it merged, the daily release auto-increments the version number